tests/core/request/test_input.py

```diff
@@ -95,6 +126,21 @@ def test_can_parse_post_params_from_url_encoded(self):
         )
         self.assertEqual(bag.get("jack"), "Daniels")
 
+    def test_can_parse_multipart_formdata_content_type(self):
+        data, content_type = encode_multipart_formdata({"key": "value", "test": 1})
+        bag = InputBag()
+        bag.load(
+            {
+                "REQUEST_METHOD": "POST",
+                "CONTENT_TYPE": content_type,
+                "CONTENT_LENGTH": str(len(data.encode("latin-1"))),
+                "wsgi.input": io.BytesIO(data.encode("latin-1")),
```

Comment:
why latin-1 here and not utf-8?

Reply:
It's just that I used an example found on internet to simulate a multipart/form-data (took me hours to find this).
But maybe it will work with utf-8, I will check.